pootle/settings/50-project.conf

```diff
@@ -38,9 +38,7 @@ MIDDLEWARE_CLASSES = [
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [
-            working_path(os.path.join('templates')),
-        ],
+        'DIRS': [],
```

Comment:
Also tested that after removing `DIRS` it works ok, so I agree with removing it.